website/newsletter/index.html

```diff
@@ -0,0 +1,68 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% set active_page = "newsletter" %}
+{% set hide_header = True %}
+{% set hide_social_footer = True %}
+{% extends "_includes/base-resp.html" %}
+
+{% block page_title %}{{ _('Join our Newsletter') }}{% endblock %}
+
+{% block header_content %}
+  <section class="text-center text-white flex justify-between pl-4 pr-4 sm:pl-8 sm:pr-8 md:pl-16 md:pr-16 max-w-7xl mx-auto pt-12 mb-20">
+    <aside class="flex-col items-start">
+      <h1 class="font-4xl md:font-hero font-thin mt-0 mb-8">{{ self.page_title() }}</h1>
+      <p class="font-lg md:font-xl tracking-wide leading-relaxed max-w-4xl lg:mb-16 mb-8 mt-0 p-links mx-auto">
+          {{ _('See exciting previews of upcoming features, read the latest desktop Thunderbird and mobile Thunderbird news, and enjoy personalized tips to protect your privacy and maximize your productivity! From the Thunderbird team directly to your inbox.') }}
+      </p>
+
+      <div class="flex flex-col items-center mb-20 mx-auto max-w-xl bg-grey p-4 rounded shadow-lg text-black-light no-underline md:font-md">
+        <div class="w-full">
```

Comment:
Add a `bg-grey-light` and a 4px border radius to this.

Then the input field and button should have a 2px border radius.